### PR TITLE
docs: add docs for overlays

### PR DESCRIPTION
**Pull request by frezbo**
Add docs for overlays.

### CODE REVIEW

File: website/content/v1.7/advanced/overlays.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+title: "Overlays"
+description: "Overlays"
+---
+
+Overlays provide a way to customize how Talos gets written down to disk during installation or upgrade.
+Overlays hooks into the Talos install steps and can be used to provide additional boot assets (in the case of single board computers),
```
Comment (smira) [inline website/content/v1.7/advanced/overlays.md:7]:
```suggestion
Overlays hook into the Talos install steps and can be used to provide additional boot assets (in the case of single board computers),
```

File: website/content/v1.7/talos-guides/install/boot-assets.md [hunk 2]
```diff
@@ -91,6 +95,64 @@ Same way upgrade process can be used to transition to a new set of system extens
...
+
+> The schematic ID is based on the schematic contents, so uploading the same schematic will return the same ID.
+
+Now we can download the metal arm64 image:
+
+* https://factory.talos.dev/image/0db665edfda21c70194e7ca660955425d16cec2aa58ff031e2abf72b7c328585/{{< release >}}/metal-arm64.raw.xz (download it and burn to a boot media)
+
+> The Image Factory URL contains both schematic ID and Talos version, and both can be changed to generate different boot assets.
+
+Once the Raspberry Pi is booted up for the first time, it will require Talos Linux `installer` image to be installed on the disk.
```
Comment (smira) [inline website/content/v1.7/talos-guides/install/boot-assets.md:134]: as it's a disk image, it doesn't need an installer actually

File: website/content/v1.7/talos-guides/install/single-board-computers/bananapi_m64.md [hunk 3]
```diff
@@ -36,16 +39,24 @@ In this example, we will assume `/dev/mmcblk0`.
...
+```yaml
+# installer-patch.yaml
+machine:
+  install:
+    image: factory.talos.dev/installer/f48c47c8a27248bc19a878c086f2cb2ed0d5aff777688af95fc88f100d1048e1:{{< release >}}
+```
 
 ```bash
-talosctl apply-config --insecure --mode=interactive --nodes <node IP or DNS name>
+talosctl apply-config --insecure --mode=interactive --config-patch=@installer-patch.yaml --nodes <node IP or DNS name>
```
Comment (smira) [inline website/content/v1.7/talos-guides/install/single-board-computers/bananapi_m64.md:59]: do we support that? :scream: 

interactive + config patch?

**Review by smira — APPROVED** [review]

File: website/content/v1.7/talos-guides/install/boot-assets.md (hunk 2 above)
Comment (frezbo) [reply website/content/v1.7/talos-guides/install/boot-assets.md:134]: ah yes, i'll add a section on upgrading if it's not already there

File: website/content/v1.7/talos-guides/install/single-board-computers/bananapi_m64.md (hunk 3 above)
Comment (frezbo) [reply website/content/v1.7/talos-guides/install/single-board-computers/bananapi_m64.md:59]: lol, i hope not, I'll fix it and drop the interactive stuff

**Comment by frezbo** [comment]
Fixes: #8474

**Pull request merged**